src/sssom/writers.py

```diff
@@ -433,10 +435,29 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
     return json_obj
 
 
+def _update_sssom_context_with_prefixmap(converter: Converter) -> str:
+    """Prepare a JSON-LD context and dump to a string."""
+    context = _load_sssom_context()
+    for k, v in dict(converter.bimap).items():
+        if isinstance(v, str):
+            if k not in context["@context"]:
```

Comment:
why do all of this checking? constructing the bimap and checking it doesn't override the built in prefix map is already done in a different place

Reply:
You are right, I copy pasted what was there be for and didn't think that it is not needed anymore.